pkg/controllers/report/utils/scanner.go

```diff
@@ -105,3 +116,29 @@ func (s *scanner) validateImages(ctx context.Context, resource unstructured.Unst
 	}
 	return &response, nil
 }
+
+func(s *scanner) ScanResourceForVAPs(ctx context.Context, resource unstructured.Unstructured, nsLabels map[string]string, policies ...v1alpha1.ValidatingAdmissionPolicy) map[v1alpha1.ValidatingAdmissionPolicy]ScanResult {
```

Comment:
Could you please check this error ? It exists in the whole file.
> invalid map key type "k8s.io/api/admissionregistration/v1alpha1".ValidatingAdmissionPolicy

Reply:
Yes, I was facing that error. But I wonder why it is throwing such an error.